participants.csv

```diff
@@ -1,7 +1,9 @@
-Zaccharie Ramzi,
-Guillaume Daval-Frérot,
-Mathilde Ripart,
-Anaïs Artiges,
-Santi,
-Tobias Liaudat,
+Zaccharie Ramzi,
+Guillaume Daval-Frérot,
+Mathilde Ripart,
+Anaïs Artiges,
+Santi,
+Tobias Liaudat,

```

Comment:
```suggestion
Zaccharie Ramzi,
Guillaume Daval-Frérot,
Mathilde Ripart,
Anaïs Artiges,
Santi,
Tobias Liaudat,
```

I think there was a problem with your editor, and it probably saved the `return` character in a weird way for all the other participants.
Maybe try to see in the settings of your editor if you can change that

Reply:
Hi Zaccharie, thanks so much for your time ! 
I'm really sorry about that, I didn't realize all the changes. As you said, I think it may have something to do with my editor. I'll check my settings !

Reply:
I think unfortunately my suggestion didn't work...
If you could try changing the settings in your editor and restoring the file with your name it would be awesome.
Also don't worry at all about the changes: you didn't do anything wrong and it's best we catch this know than in the middle of a real contribution. This is a testing ground so don't hesitate to try stuff, you won't break anything.